cardano-config/src/Cardano/TracingOrphanInstances/Shelley.hs

```diff
@@ -0,0 +1,336 @@
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE LambdaCase            #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TypeFamilies          #-}
+{-# LANGUAGE UndecidableInstances  #-}
+{-# LANGUAGE NamedFieldPuns        #-}
+{-# LANGUAGE EmptyCase             #-}
+{-# LANGUAGE StandaloneDeriving    #-}
+{-# LANGUAGE DerivingStrategies    #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans  #-}
+{-# OPTIONS_GHC -Wno-all-missed-specialisations #-}
```

Comment:
Since we dropped `-Weverything` for `-Wall` that should not be needed.

Reply:
Yes. I've not added or removed these in these recent Shelley plumbing PRs.

There's over 20 of these remaining in the repo, in both `cardano-config` and `cardano-node`. I suggest we remove them all as part of a clean-up PR.